cmd/collectors/zapi/plugins/svm/svm.go

```diff
@@ -508,10 +515,17 @@ func (my *SVM) GetSSHData() (map[string]string, error) {
 		return nil, errs.New(errs.ErrNoInstance, "no records found")
 	}
 
+	compile, err := regexp.Compile("(.*)_cbc.*")
+	if err != nil {
+		my.Logger.Error().Err(err).Msg("Failed to compile regex: (.*)_cbc.*")
+	}
+
 	for _, sshData := range result {
 		svmName := sshData.GetChildContentS("vserver-name")
 		sshList := sshData.GetChildS("ciphers").GetAllChildContentS()
-		sshMap[svmName] = strings.Join(sshList, ",")
+		ciphersVal := strings.Join(sshList, ",")
```

Comment:
do we need a deterministic order for the joined list? Typically, we sort before joining

Reply:
we don't have that sorted order use-case for this, we just need list of supported ciphers and other boolean value(insecured) if it has `_cbc` weak ciphers.

Reply:
sorted the list before joining